src/Kernel/Magic/SimulateNoise.cs

```diff
@@ -0,0 +1,137 @@
+
+                    The behavior of this magic command can be controlled through the `%experimental.noise_model` magic command,
+                    and the `opensim.nQubits` and `opensim.representation` configuration settings.
+                ".Dedent(),
+                Examples = new string[]
+                {
+                    @"
+                        Simulate a Q# operation defined as `operation MyOperation() : Result`:
+                        ```
+                        In []: %simulate MyOperation
+                        Out[]: <return value of the operation>
+                        ```
+                    ".Dedent(),
+                    @"
+                        Simulate a Q# operation defined as `operation MyOperation(a : Int, b : Int) : Result`:
+                        ```
+                        In []: %simulate MyOperation a=5 b=10
+                        Out[]: <return value of the operation>
+                        ```
+                    ".Dedent(),
+                }
+            })
+        {
+            this.SymbolResolver = resolver;
+            this.ConfigurationSource = configurationSource;
+            this.Logger = logger;
+            this.NoiseModelSource = noiseModelSource;
+
+            if (engine is IQSharpEngine iQSharpEngine)
+            {
+                iQSharpEngine.RegisterDisplayEncoder(new MixedStateToHtmlDisplayEncoder());
+            }
+        }
+
+        /// <summary>
+        ///      The symbol resolver used by this magic command to find
+        ///      operations or functions to be simulated.
+        /// </summary>
+        public ISymbolResolver SymbolResolver { get; }
+
+        /// <summary>
+        ///     The configuration source used by this magic command to control
+        ///     simulation options (e.g.: dump formatting options).
+        /// </summary>
+        public IConfigurationSource ConfigurationSource { get; }
+
+        /// <inheritdoc />
+        public override ExecutionResult Run(string input, IChannel channel) =>
+            RunAsync(input, channel).Result;
+
+        /// <summary>
+        ///     Simulates an operation given a string with its name and a JSON
+        ///     encoding of its arguments.
+        /// </summary>
+        public async Task<ExecutionResult> RunAsync(string input, IChannel channel)
+        {
+            var inputParameters = ParseInputParameters(input, firstParameterInferredName: ParameterNameOperationName);
+
+            var name = inputParameters.DecodeParameter<string>(ParameterNameOperationName);
+            var symbol = SymbolResolver.Resolve(name) as dynamic; // FIXME: Should be IQSharpSymbol.
+            if (symbol == null) throw new InvalidOperationException($"Invalid operation name: {name}");
+
+            var qsim = new OpenSystemsSimulator(
+                ConfigurationSource.OpenSystemsSimulatorCapacity //,
+                // ConfigurationSource.OpenSystemsSimulatorRepresentation
```

Comment:
Is this functionality that is not ready yet or a hold-over from a previous design?

Reply:
Ah, this was due to a cross-repo change in the API not having propagated correctly; that should be fixed now such that I can uncomment. Thanks for catching this!